src/gluonts/dataset/loader.py

```diff
@@ -37,6 +50,63 @@ def __call__(self, data, is_train):
         yield from batcher(data, self.batch_size)
 
 
+class Stack(Transformation, BaseModel):
+    def __call__(self, data, is_train):
+        for batch in data:
+            yield rows_to_columns(batch, np.array)
```

Comment:
There is an argument that we don't need to wrap this in an `np.array`.

However, `torch.tensor` and `mx.ndarray` like to work with `np.array` instances rather than lists.

Reply:
You could wrap as numpy array in a separate step using `ValMap`? But then again, it's also fine like this

Reply:
I guess we can leave it as it is.